src/content/troubleshooting/_index.en.md

```diff
@@ -16,7 +16,7 @@ The guide has been broken into different sections for easy navigation.
 ### Pre-requisite
 Before we begin troubleshooting, run `subctl version` to obtain which version of the Submariner components you are running.
 
-Run `kubectl get services | grep <service-name>` to get information about the service you're trying to access. This will provide you with the Service *Name*, *Namespace* and *ServiceIP*. If **Globalnet** is enabled, you will also need the *globalIp* of the service by running
+Run `kubectl get services -n <service-namespace> | grep <service-name>` to get information about the service you're trying to access. This will provide you with the Service *Name*, *Namespace* and *ServiceIP*. If **Globalnet** is enabled, you will also need the *globalIp* of the service by running
```

Comment:
nit: Our docs use a convention of "GlobalNet". It would be nice to follow the same convention everywhere.